### PR TITLE
jsDelivr related changes

### PR DESCRIPTION
**Pull request by jimaek**
- Migrated privacy policy to jsDelivr
- Removed jobs
- Switched GA
- Removed some affiliate stuff

### CODE REVIEW

**Comment by jimaek** [comment]
@XhmikosR Sorry I just noticed the related changes over here https://github.com/MaxCDN/bootstrapcdn/pull/1675
Let me know what you want to do. I could remove my test changes from here

File: app.js [hunk 1]
```diff
@@ -132,9 +132,7 @@ app.use('/bootswatch4/', routes.bootswatch4Route);
 app.use('/data/bootstrapcdn.json', routes.dataRoute);
 app.use('/fontawesome/', routes.fontawesomeRoute);
 app.use('/integrations/', routes.integrationsRoute);
-app.use('/jobs/', routes.jobsRoute);
```
Comment (XhmikosR) [inline app.js:135]: I'd add redirects for the removed pages

File: views/_partials/footer.pug [hunk 2]
```diff
@@ -27,6 +22,6 @@ footer.mt-5.py-4.text-center.border-top
                         a(href=friend.url, rel=`noopener${friend.nofollow !== false ? " nofollow" : ""}`, target='_blank')=friend.name 
 
                 p.
-                    #[a(href='/about/') About] · #[a(href='/privacy-policy/') Privacy Policy]
+                    #[a(href='/about/') About] · #[a(href='https://www.jsdelivr.com/terms/privacy-policy-jsdelivr-net',  target='_blank') Privacy Policy]
```
Comment (XhmikosR) [inline views/_partials/footer.pug:25]: Please add `rel='noopener'` and remove the stray whitespace

File: views/_partials/jumbotron.pug [hunk 3]
```diff
@@ -2,23 +2,21 @@
...
+        | The recommended CDN
         |  for 
         a(href='https://getbootstrap.com/', rel='noopener nofollow', target='_blank') Bootstrap
         | , 
         a(href='https://fontawesome.com/', rel='noopener nofollow', target='_blank') Font Awesome
         |  and 
         a(href='https://bootswatch.com/', rel='noopener nofollow', target='_blank') Bootswatch
-        | .
+        |  powered by 
+        a(href='https://www.jsdelivr.com', target='_blank') jsDelivr
```
Comment (XhmikosR) [inline views/_partials/jumbotron.pug:13]: Same here

**Comment by XhmikosR** [comment]
@jimaek yeah, I'd prefer if the test changes were made in #1675.

BTW the content types should match what we had. Personally, I prefer being spec-compliant and we've gone to great lengths all these year to do this.

**Comment by jimaek** [comment]
I made the changes, but I guess the tests PR needs to get merged first to get the real test results here.

**Comment by XhmikosR** [comment]
The CDN tests yeah. I think better finish with #1675 first.

File: views/_partials/jumbotron.pug [hunk 4]
```diff
@@ -2,23 +2,21 @@
     h1.display-5 BootstrapCDN
 
     p.lead
-        | The recommended 
-        a(href='https://www.stackpath.com/products/cdn/?utm_campaign=Partner%20Display&utm_source=Partner%20Display&utm_medium=bootstrapcdn', rel='noopener', target='_blank') CDN
+        | The recommended CDN
```
Comment (XhmikosR) [inline views/_partials/jumbotron.pug:5]: Not sure if you want to link to jsdevilr here too or maybe https://en.wikipedia.org/wiki/Content_delivery_network? It's nitpicking, so it's your call :)

Comment (jimaek) [reply views/_partials/jumbotron.pug:5]: Yeah maybe in the future. For now let's just do a cleanup :)

File: app.js [hunk 5]
```diff
@@ -124,17 +124,15 @@ app.locals.semver = semver;
...
 app.use('/books/', routes.booksRoute);
 app.use('/bootlint/', routes.bootlintRoute);
 app.use('/bootswatch/', routes.bootswatchRoute);
 app.use('/bootswatch4/', routes.bootswatch4Route);
 app.use('/data/bootstrapcdn.json', routes.dataRoute);
 app.use('/fontawesome/', routes.fontawesomeRoute);
 app.use('/integrations/', routes.integrationsRoute);
-app.use('/jobs/', routes.jobsRoute);
 app.use('/legacy', routes.legacyRoutes);
-app.use('/privacy-policy/', routes.privacyPolicyRoute);
```
Comment (XhmikosR) [inline app.js:137]: This redirect is missing @jimaek

**Comment by XhmikosR** [comment]
The Heroku preview does not fire, and I can't seem to be able to manually trigger it...

File: config/helmet-csp.js [hunk 6]
```diff
@@ -32,14 +30,14 @@ const CSP = {
...
+        'syndication.twitter.com',
         'stats.g.doubleclick.net',
         'ad.doubleclick.net',
         '*.convertro.com',
         '*.c3tag.com',
         '*.2mdn.net',
         'launchbit.com',
-        'www.launchbit.com',
-        'www.ziprecruiter.com'
+        'www.launchbit.com'
```
Comment (XhmikosR) [inline config/helmet-csp.js:40]: We should check if the other entries are all needed, but I don't have the time to do it now.

Comment (jimaek) [reply config/helmet-csp.js:40]: Lets do things in small and quick steps. As we remove more pages we can review this and update it too.
I think this PR is ready and after that develop can be merged into production.

Comment (XhmikosR) [reply config/helmet-csp.js:40]: Sure, just wanted to note the issue :P

**Comment by XhmikosR** [comment]
The CDN has some issues that probably stem from the origin... I'll restart the run, but it seems that it's super slow when hitting the origin or something like that.

>         https://stackpath.bootstrapcdn.com/bootstrap/4.5.1/css/bootstrap-reboot.css
>           it works:
>     Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/runner/work/bootstrapcdn/bootstrapcdn/test/functional_test.js)

**Review by XhmikosR — APPROVED** [review]

**Comment by jimaek** [comment]
> The CDN has some issues that probably stem from the origin... I'll restart the run, but it seems that it's super slow when hitting the origin or something like that.
> 
> > ```
> >     https://stackpath.bootstrapcdn.com/bootstrap/4.5.1/css/bootstrap-reboot.css
> >       it works:
> > Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/runner/work/bootstrapcdn/bootstrapcdn/test/functional_test.js)
> > ```

Yeah uncached will be slow unfortunately(but not 10s). We utilize an additional layer between CF and S3 because CF cant handle S3 easily by itself. Thats why I wanted to review the slowness separately after we finish changing stuff and purging the cache.

**Pull request merged**